test_runner/src/main/kotlin/ftl/util/SystemOutColor.kt

```diff
@@ -3,6 +3,7 @@ package ftl.util
 enum class SystemOutColor(val ansiCode: String) {
     DEFAULT("\u001B[0m"),
     RED("\u001B[31m"),
+    BLUE("\u001B[34m"),
```

Comment:
I known you add only blue color but probably ansiCode can be private.

Reply:
cannot be cause it is used later in printing informations